cli/dcoscli/task/main.py

```diff
@@ -161,6 +174,156 @@ def _log(follow, completed, lines, task, file_):
     return 0
 
 
+def get_nested_container_id(task):
+    """ Get the nested container id from mesos state.
+
+    :param task: task definition
+    :type task: dict
+    :return: comma separated string of nested containers
+    :rtype: string
+    """
+    assert isinstance(task, dict)
+
+    # get current task state
+    task_state = task.get('state')
+    if not task_state:
+        raise DCOSException('Invalid executor info. '
+                            'Missing field `state`')
```

Comment:
Can we give a little more context for the user? Should we put the full task output in cli debug mode?